third_party/SQLite/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 
 IF ( MSVC ) # visual c++ (VS 2013)
     ADD_DEFINITIONS("-W0")
+    ADD_CXX_DEBUG_DEFINITIONS("/fp:fast") # override fp:strict in debug builds
```

Comment:
Good find here, I don't exactly understand why this is needed for SQLite, but I'm glad you got this worked out so it will build properly and catch NaNs for you.

Reply:
Yeah, with fp:strict, the compiler chokes on this block, complaining "initializer is not a constant", starting at line 20910 in SQLite.cc.
```
static const struct {
  u8 eType;           /* Transformation type code */
  u8 nName;           /* Length of th name */
  char *zName;        /* Name of the transformation */
  double rLimit;      /* Maximum NNN value for this transform */
  double rXform;      /* Constant used for this transform */
} aXformType[] = {
  { 0, 6, "second", 464269060800.0, 86400000.0/(24.0*60.0*60.0) },
  { 0, 6, "minute", 7737817680.0,   86400000.0/(24.0*60.0)      },
  { 0, 4, "hour",   128963628.0,    86400000.0/24.0             },
  { 0, 3, "day",    5373485.0,      86400000.0                  },
  { 1, 5, "month",  176546.0,       30.0*86400000.0             },
  { 2, 4, "year",   14713.0,        365.0*86400000.0            },
};
```
I guess the alternative would be to change these, but I didn't want to touch that.

Reply:
Yeah I wouldn't touch sqlite code.